nucypher/utilities/sandbox/constants.py

```diff
@@ -129,6 +129,8 @@ def select_test_port() -> int:
 
 MOCK_ALLOCATION_REGISTRY_FILEPATH = os.path.join(BASE_TEMP_DIR, f'{BASE_TEMP_PREFIX}test-allocation-registry-{str(datetime.now().strftime(DATETIME_FORMAT))}.json')
 
+MOCK_INDIVIDUAL_ALLOCATION_FILEPATH = os.path.join(BASE_TEMP_DIR, f'{BASE_TEMP_PREFIX}test-individual-allocation-{str(datetime.now().strftime(DATETIME_FORMAT))}.json')
```

Comment:
I don't think `str()` is needed here - `strftime()` returns a string anyway

Reply:
✅